src/pygrambank/cldf.py

```diff
@@ -215,7 +215,7 @@ def coders(sheet, row):
             'Newick': glottolog.api.newick_tree(
                 gc, template='{l.id}', nodes=glottolog.languoids_by_glottocode),
         } for gc in families], key=lambda d: d['ID'])
-    dataset.write(**data)
+    dataset.write(zipped=['ValueTable'], **data)
```

Comment:
Ah, we don't really use `cldfbench.CLDFWriter` here. We might consider changing that.